src/main/java/com/github/dockerjava/core/DockerClientConfig.java

```diff
@@ -102,8 +102,9 @@ private URI checkDockerHostScheme(URI dockerHost) {
     private String checkDockerCertPath(boolean dockerTlsVerify, String dockerCertPath) {
         if (dockerTlsVerify) {
             if (StringUtils.isEmpty(dockerCertPath)) {
-                throw new DockerClientException(
-                        "Enabled TLS verification (DOCKER_TLS_VERIFY=1) but certifate path (DOCKER_CERT_PATH) is not defined.");
+                return dockerCertPath;
+//                throw new DockerClientException(
```

Comment:
As I said: I really don't like this. This is bad design in my eyes. I will experiment locally with interface and different impl. I think only the default config impl needs a builder. If someone wants a different configuration implementation its his / her reponsibility to create the instance, maybe even with a custom builder. WDYT?